worker/transcribee_worker/reencode.py

```diff
@@ -46,31 +45,18 @@ def work(_):
             progress="-",
             map_metadata="-1",
             **output_params,
-        ).run_async(pipe_stdout=True, pipe_stderr=True)
+        ).run_async(pipe_stderr=True, pipe_stdout=True)
+        assert cmd.stderr
         assert cmd.stdout
-        raw_line: bytes
-        progress_dict = {}
 
-        stderr_data = []
+        Thread(target=read_progress, args=(cmd.stdout,)).start()
 
-        for pipe, raw_line in readlines([cmd.stdout, cmd.stderr]):
-            if pipe == cmd.stderr:
-                stderr_data.append(raw_line.decode())
-                continue
-            if b"=" not in raw_line:
-                continue
-            key, value = raw_line.decode().strip().split("=", maxsplit=1)
-            progress_dict[key] = value.strip()
+        stderr_data = []
+        for raw_line in cmd.stderr:
+            line = raw_line.decode().rstrip()
+            logging.info(line)
```

Comment:
Why logging? This way we never see it in the logs sent to the db and can only investigate issues if we have access to the worker logs, which is not guaranteed

Reply:
Hm? stderr is still passed to the extra_data. The logging is mainly for local development.